packages/database/prisma/schema.prisma

```diff
@@ -326,3 +327,17 @@ model PurchaseUserTransfer {
   confirmedAt   DateTime?
   completedAt   DateTime?
 }
+
+
+model Bookmark {
+  id            String                    @id @default(uuid())
+  userId        String 
+  user          User                      @relation(fields: [userId], references: [id], onDelete: Cascade)
+  resourceId    String
+  type          String
+  fields        Json                      @default("{}")
+  createdAt     DateTime                  @default(now())
+  updatedAt    DateTime                   @updatedAt() @default(now())
+
+  @@index([userId, resourceId])
```

Comment:
added an index on `userId` and `resourceId` as that will be the primary way we query this table